packages/scrollable-image/ui/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const path = require('path')
+
+const app = express()
+
+app.use(express.static('public'))
+
+app.get('/', (req, res) => {
+  res.status(200).sendFile(path.join(__dirname, '/index.html'))
+})
+
+module.exports = {
+  app,
+}
```

Comment:
Should we commit this file in `cloud-functions` repo?

Reply:
For the sake of private service account, it should be committed into `cloud-functions` repo.